go.mod

```diff
@@ -38,6 +38,7 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/reso
 
 replace (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver => github.com/amazon-contributing/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver v0.0.0-20241104203805-20919412150d
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightskueuereceiver => github.com/amazon-contributing/opentelemetry-collector-contrib/receiver/awscontainerinsightskueuereceiver v0.0.0-20241112181539-bc8278558352
```

Comment:
let's get this in aws-cwa-dev

Reply:
Done, see [this PR](https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/253).

Reply:
You need to update these versions for all components post the merge to aws-cwa-dev.

Reply:
do you need this line? `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightskueuereceiver` doesnt exist upstream

Reply:
Versions updated.

I believe that line is necessary as the new module in our fork of the OTel collector repo is called `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightskueuereceiver`.